docs/coordinates/matchsep.rst

```diff
@@ -266,35 +266,35 @@ matches in ``catalog`` around any locations in ``c``.  Hence, indices into both
 These can then be indexed back into the two |skycoord| objects, or, for that
 matter, any array with the same order::
 
-    >>> np.all(c[idxc].separation(catalog[idxcatalog]) == d2d)  # doctest: +SKIP
+    >>> np.all(c[idxc].separation(catalog[idxcatalog]) == d2d)
     True
-    >>> np.all(c[idxc].separation_3d(catalog[idxcatalog]) == d3d)  # doctest: +SKIP
+    >>> np.all(c[idxc].separation_3d(catalog[idxcatalog]) == d3d)
     True
-    >>> print catalog_objectnames[idxcatalog]  # doctest: +SKIP
+    >>> print catalog_objectnames[idxcatalog]
     ['NGC 1234' 'NGC 4567' ...]
 
 Note, though, that this dual-indexing means that ``search_around_*`` does not
 work well if one of the coordinates is a scalar, because the returned index
 would not make sense for a scalar::
 
-    >>> scalarc = SkyCoord(1*u.deg, 2*u.deg)  # doctest: +SKIP
-    >>> idxscalarc, idxcatalog, d2d, d3d = catalog.search_around_sky(scalarc, 1*u.deg)  # THIS DOESN'T ACTUALLY WORK  # doctest: +SKIP
-    >>> scalarc[idxscalarc]  # doctest: +SKIP
```

Comment:
keep these for PEP8. Btw. the description a few lines above says why this is skipped. It's always good to check the context of the changes rather than do a find-replace.

Reply:
Gotcha!